Re-Enable federated queries

**Describe the bug**

Federated queries currently give errors like
```Unknown error: Service URI https://query.wikidata.org/sparql is not allowed```

**Expected behavior**

Federated query runs for endpoints that are included in some kind of whitelist/ allowlist that should contain Wikidata.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to https://portal.mardi4nfdi.de/w/index.php?title=User:Daniel/Queries/Federated&oldid=5810775 
2. Try any of the queries in the "MaRDI and" sections, e.g. [this one](https://query.portal.mardi4nfdi.de/#PREFIX%20wdt%3A%20%3Chttps%3A%2F%2Fportal.mardi4nfdi.de%2Fprop%2Fdirect%2F%3E%0APREFIX%20wd%3A%20%3Chttps%3A%2F%2Fportal.mardi4nfdi.de%2Fentity%2F%3E%0A%0APREFIX%20wikidata_wd%3A%20%3Chttp%3A%2F%2Fwww.wikidata.org%2Fentity%2F%3E%0APREFIX%20wikidata_wdt%3A%20%3Chttp%3A%2F%2Fwww.wikidata.org%2Fprop%2Fdirect%2F%3E%0A%0ASELECT%20%2a%20WHERE%20%7B%0A%20%20%3Fitem%20wdt%3AP676%20%3FzbMATH_author%20.%0A%20%20FILTER%20NOT%20EXISTS%20%7B%20%3Fitem%20wdt%3AP12%20%3FWikidataID%20.%20%7D%0A%20%20%3Flink%20schema%3Aabout%20%3Fitem%20.%20%0A%20%20%3Flink%20schema%3AinLanguage%20%22en%22%20.%0A%20%20%20%20service%20%3Chttps%3A%2F%2Fquery.wikidata.org%2Fsparql%3E%20%7B%0A%20%20%20%20%20%20%3Fwd%20wikidata_wdt%3AP1556%20%3FzbMATH_author%20.%0A%20%20%20%7D%0A%7D%0ALIMIT%205000)
3. See error

**Screenshots**

![Screenshot 2024-01-25 152012](https://github.com/MaRDI4NFDI/portal-compose/assets/465923/76ac948e-1c09-42c2-a887-7f6b2a470718)

**Additional context**

For Wikidata, the procedure is described at https://wikitech.wikimedia.org/wiki/Wikidata_Query_Service#Updating_federation_allowlist .

**Checklist for this issue**: 
(Some checks for making sure this issue is completely formulated)
- [x] Assignee has been set for this issue 
- [X] All fields of the issue have been filled 
- [X] the main MaRDI project has been assigned as project

The query still does not work as it runs into wmde rate limits. But at least it is a different error
```
Caused by: java.io.IOException: Broken pipe

	at sun.nio.ch.FileDispatcherImpl.write0(Native Method)

	at sun.nio.ch.SocketDispatcher.write(SocketDispatcher.java:47)

	at sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:93)

	at sun.nio.ch.IOUtil.write(IOUtil.java:51)

	at sun.nio.ch.SocketChannelImpl.write(SocketChannelImpl.java:471)

	at org.eclipse.jetty.io.ChannelEndPoint.flush(ChannelEndPoint.java:264)

	... 37 common frames omitted

19:16:41.164 [qtp861842890-37] INFO  o.w.q.r.b.t.ThrottlingFilter - A request is being throttled.
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -237,6 +237,7 @@ services:
       - WIKIBASE_SCHEME=${WIKIBASE_SCHEME:-https}
       - WDQS_HOST=wdqs.svc
       - WDQS_PORT=9999
+      - ALLOWLIST=whitelist.txt
       - BLAZEGRAPH_OPTS='-Dorg.wikidata.query.rdf.tool.rdf.RdfRepository.timeout=3600'
     expose:
       - 9999

